src/main/resources/org/jenkinsci/plugins/github/config/GitHubPluginConfig/config.groovy

```diff
@@ -5,15 +5,12 @@ import com.cloudbees.jenkins.GitHubPushTrigger
 def f = namespace(lib.FormTagLib);
 
 f.section(title: descriptor.displayName) {
-    f.entry(title: _("Servers configs with credentials to manage GitHub integrations"),
-            description: _("List of GitHub Servers to manage hooks, set commit statuses etc."),
+    f.entry(title: _("GitHub Servers"),
             help: descriptor.getHelpFile()) {
         
         f.repeatableHeteroProperty(
                 field: "configs",
-                hasHeader: "true",
-                addCaption: _("Add GitHub Server Config"),
```

Comment:
Maybe just set to Add GitHub Server? As of its better for end-users than simple "Add" (it adds a userfriendly context)